### PR TITLE
First pass fixes for quickstart demo

### PR DESCRIPTION
**Pull request by m4sterbunny**
- fixes typos and duplicated words
- standardizes components to proper nouns (Verifier, Issuer, Identity Holder)
- simplifies sentence structure
- improves UI through use of tabs

Doesn't address:
- inconsistent use of concatenation (Doesn't vs does not)
- white space in screenshots that could be shaved

Possible additional improvements possible:
- would like to play with numbering for tutorial flow

### CODE REVIEW

**Comment by m4sterbunny** [comment]
@0xpulkit a quick pass to fix some typos/sentence flow, plus some structural changes for you to consider. Great to meet you today.

File: docs/quick-start-demo.md [hunk 1]
```diff
@@ -15,98 +15,133 @@ keywords:
...
-- JSON schema URL
-  `ipfs://QmTSwnuCB9grYMB2z5EKXDagfChurK5MiMCS6efrRbsyVX`
-- JSON-LD Context
-  `ipfs://QmdH1Vu79p2NcZLFbHxzJnLuUHJiMZnBeT7SNpLaqK7k9X`
+We will use <a href="https://issuer-demo.privado.id/">the Issuer Node UI Demo environment</a> to manage credentials. This environment enables an Issuer to create and manage identities and credentials, generate connections, and manage schemas.
 
 :::note
 
-To learn how to set up your own issuer environment by deploying an issuer node, visit the [<ins>Issuer section in the documentation</ins>](./issuer/issuer-overview.md).
+When an Issuer releases a new credential type, they create a schema for that credential. This requires a set of JSON key pair data or files that gather all the attributes of that specific credential.
```
Comment (amonsosanz) [inline docs/quick-start-demo.md:78]: > a set of JSON key pair data or files

"JSON" and "key pair data" are redundant. (I guess you mean "key/**value** pair data")

I'd leave something closer to the original:

> When an Issuer releases a new credential type, they create a schema for that credential (i.e. a set of JSON files that gather all the attributes of that specific credential).

File: docs/quick-start-demo.md [hunk 2]
```diff
@@ -15,98 +15,133 @@ keywords:
...
 </div>
 
-4. After you click on **Create Credential Link**, you'll be presented with a Universal Link and a Deep Link to the credential offer, along with a QR code configured for these links. The QR code can be scanned directly with the Privado ID Wallet app.
+4. Click on **Create Credential Link** to be presented with a Universal Link and a Deep Link to the credential offer, along with a QR code configured for these links. The QR code can be scanned directly with the Privado ID Wallet app, that is, the Mobile App.
 
     :::info
 
-    When the user interacts with the [<ins>Universal Link</ins>](/docs/wallet/universal-links.md) , it will launch the Privado ID Web Wallet in the browser or Privado ID Wallet app in case of a mobile phone displaying the credential offer to claim the credential. Deep link could be handled only by mobile wallet app though.
+    When the user interacts with the [<ins>Universal Link</ins>](/docs/wallet/universal-links.md), it will launch the Privado ID Web Wallet in the browser or Privado ID Wallet app in case of a mobile phone displaying the credential offer to claim the credential. The Deep link supports the Mobile Wallet App.
```
Comment (amonsosanz) [inline docs/quick-start-demo.md:144]: > The Deep link supports the Mobile Wallet App.

Here we lose the concept that Deep Links can only be used from mobile applications and not in the Web Wallet.

Comment (m4sterbunny) [reply docs/quick-start-demo.md:144]: I will try again to simplify this, but retain the data

Comment (m4sterbunny) [reply docs/quick-start-demo.md]: PTAL @ lines 144-148 that now cover this @amonsosanz WDYT?

File: docs/quick-start-demo.md (hunk 1 above)
Comment (m4sterbunny) [reply docs/quick-start-demo.md:78]: I wasn't convinced that files plural was correct base on what I had seen; seemed more likely it was data (plural) to create schema (singular) --> tried an approach that side steps both issues PTAL

Comment (m4sterbunny) [reply docs/quick-start-demo.md]: ah yes, ok plural returned

File: docs/quick-start-demo.md (hunk 2 above)
Comment (amonsosanz) [reply docs/quick-start-demo.md]: Not yet precise I'd say...

`Deep Link supports the Mobile Wallet App only` should be:
`Deep Links are supported by the Privado ID Mobile App Wallet and other mobile wallets`

* Deep links do not support wallets, but the other way around.
* Not only our mobile wallet can use these, but any mobile wallet.

`The Universal Link supports the Web App only` should be
`The Universal Links are supported by both the Web Wallet and the Privado ID Mobile App Wallet. They might also be supported by third-party mobile wallets`

* Universal links do not support wallets, but the other way around.
* Our web and mobile wallets and other mobile wallets might also support these.

All this is a bit tricky 😅

Let's also make sure I'm right checking it with @0xpulkit.

Comment (m4sterbunny) [reply docs/quick-start-demo.md]: Thanks @amonsosanz did another pass PTAL.

Comment (amonsosanz) [reply docs/quick-start-demo.md]: Now it's perfect! Thx to you.

**Review by amonsosanz — APPROVED** [review]

**Comment by m4sterbunny** [comment]
Thanks for the review @amonsosanz --> I don't have merge permissions, can you do it?

**Comment by gseriche** [comment]
Done!

**Pull request merged**